src/main/java/edu/bu/cs633/minimalsearchengine/utils/Utilities.java

```diff
@@ -0,0 +1,39 @@
+package edu.bu.cs633.minimalsearchengine.utils;
+
+import edu.bu.cs633.minimalsearchengine.exceptions.customExceptions.ConstraintViolationException;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public final class Utilities {
+
+    public static final Set<String> STOP_WORDS = new HashSet<>(Arrays.asList("i", "me", "my", "myself", "we",
```

Comment:
Are stop words checked before or after input sanitation? I assume after, in which case the apostrophe (') character should never make it this far, so the contractions here ("it's", "you've") should be modified to however we end up handling sanitizing input. If, for example, an input of "you've" becomes "youve" then that should be the stop word. If "you've" instead becomes "you" and "ve" then that is a different handling.

Reply:
@sargent-bu I thought about the STOP_WORDS handling as well. I have not decided to how to use/handle the query from user containing (you've). That's what I was planning on bringing up to the others this Tuesday, but at least for sure we will use this set of words as stop words, and that's why I added them as part of this PR.